### ISSUE TITLE
Custom Date Format is not applied in the preview field datetime picker window

### ISSUE THREAD

**Issue by Irajjj** [dotnetcore/BootstrapBlazor#258]
### Describe the bug
When I choose a custom date format, datetime format is shown correctly in the main textbox but in the preview on top of the datetime picker window it is still shown in default format which is American format.   
I wish that preview date is also shown in the format that I have chosen, like dd.MM.yyyy not the default MM.dd.yyyy.

![datetime picker format](https://user-images.githubusercontent.com/35769902/194715555-978cb13c-0aa8-4bcd-960d-a8e8917aae67.jpg)

### To Reproduce

```
<div class="col-12 mt-3">
<DateTimePicker TValue="DateTime" ShowLabel="true" Format="dd.MM.yyyy HH:mm:ss" DisplayText="Datetime" @bind-Value="DateTime1" ViewMode="DatePickerViewMode.DateTime" />
</div>
```

### Further technical details
- BootstrapBlazor Version=6.6.7
- net6.0
- The IDE: VS 2022

**Comment by ArgoZhang** [comment]
@Irajjj sorry for the delayed reply. thanks for contacting us. It will be fixed in the next release

**Comment by Irajjj** [comment]
@ArgoZhang Thank you for your reply. Can I ask when I can expect this to be fixed? It would be great if it can be fixed in a short time, because I need it for a release and my deadline is near. I'll appreciate it.

**Comment by ArgoZhang** [comment]
@Irajjj maybe in the three hours. have a nice day. i am working on it

**Comment by ArgoZhang** [comment]
@Irajjj it has been fixed. I notice that you provide the package version as `6.6.7`. the `BootstrapBlazor` latest version is `6.11.3`.

so I rollback to `6.6.7` then fixed it and release the new version `6.6.8-beta01`. is right? do you have a try this `6.6.8-beta01` version? please let me know if you have any questions about this.

**Comment by ArgoZhang** [comment]
@Irajjj thanks. it's has been merged into the latest version

**Comment by Irajjj** [comment]
Thank you again.
Can you tell me in which latest version the fix for time format is merged? I checked several of recent versions, non of them worked or was stable. I checked 6.11.8-beta01,  6.11.8,  6.11.7 and 6.11.0.
So I came back to use 6.6.8-beta01 and it works.

**Comment by ArgoZhang** [comment]
@Irajjj I recheck it. it's working fine with 6.11.5 and above. finally released v6.11.8 you can try it. please let me know if you have any questions
